templates/terms_and_conditions.html

```diff
@@ -0,0 +1,45 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="Paper-copy">
+      <p>NurseConnect is a National Department of Health (NDoH) initiative. It is the one stop shop where nurses can find information, ask for expert help, receive emotional support and have a line of direct communication with the National Department of Health. It utilises mobile software to give nurses in South Africa access to clinical, psychosocial and learning content. </p>
+
+      <p>NurseConnect sends messages over WhatsApp and SMS to help nurses with the important work they do. The NurseConnect mobisite, nurseconnect.org, offers a content repository that can be accessed by nurses at any time. There is also a NurseConnect Helpdesk. Nurses can reply to any NurseConnect message over WhatsApp or SMS with a question and a professional will reply. </p>
+
+      <h3>What is the Refer-a-Friend campaign?</h3>
+
+      <p>The Refer-a-Friend campaign is a drive to recruit more nurses to sign up to NurseConnect.</p>
+
+      <p>It runs as follows: </p>
+
+      <p>There are two registration mechanisms:</p>
+
+      <p>a. As a nurse registered with NurseConnect, you can register another nurse via the USSD line (*135*550*5#). For every nurse you sign up, you will receive R10 worth of airtime as an incentive (‘the  airtime incentive”) loaded into your prepaid or top-up plan mobile device; alternatively</p>
+
+      <p>b. As a nurse registered with NurseConnect, you can send a link to other nurses who have not yet registered with NurseConnect. Those nurses can access the link and register directly for NurseConnect. You will receive the airtime incentive loaded into your prepaid or top-up plan mobile device, for each and every nurse that you refer who signs up to NurseConnect up to the capped amount.</p>
```

Comment:
This style of list can also be an ordered list, ie `<ol type="a">`

Reply:
thanks for the tip